test/acceptance/build/slack_test.js

```diff
@@ -187,113 +187,240 @@ describe('Acceptance | Build | Slack', function () {
             },
           });
         });
-
-        it('returns the confirmation modal with a warning', async function () {
-          // given
-          const tagNock = nock('https://api.github.com')
-            .get('/repos/github-owner/github-repository/tags')
-            .twice()
-            .reply(200, [
-              {
-                commit: {
-                  url: 'https://api.github.com/repos/github-owner/github-repository/commits/1234',
+        describe('when config file has been changed', function () {
+          it('returns the confirmation modal with a warning and sends a slack message to tech-releases', async function () {
```

Comment:
Splitter le test en deux, il devient monstrueux
```suggestion
          it('sends a slack message to tech-releases', async function () {
```